website_google_tag_manager/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 ABF OSIELL <http://osiell.com>
+# Copyright 2018 Tecnativa - Cristina Martin R.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Google Tag Manager",
+    "version": "11.0.1.0.0",
+    "author": "ABF OSIELL, Tecnativa, Odoo Community Association (OCA)",
```

Comment:
@cristinamartinrod Add `website` key

Reply:
Hi Nikul, on "name"?

Reply:
No Dear, `website: https://github.com/OCA/website `

Added `website` key in manifest

Reply:
Done